### PR TITLE
Improved interface

### PR DESCRIPTION
**Pull request by harshitaaaaaa**

### CODE REVIEW

File: Projects/Whatsapp/css/style.css [hunk 1]
```diff
@@ -196,29 +235,46 @@
...
-#attatchments,
-#menu2,
-#mic{
-    height: 20px;
+#message, #menu, #menu2{
+    height: 27.5px;
     padding: 1px;
     margin-top: 5px;
     padding: 0 5px;
+    padding-left: 6px;
```
Comment (harshitaaaaaa) [inline Projects/Whatsapp/css/style.css:250]: Please review this commit.

File: Projects/Whatsapp/css/style.css [hunk 2]
```diff
@@ -196,29 +235,46 @@
     cursor: pointer;
 }
 
+#status{
+    height: 25px;
+    padding: 1px;
+    margin-top: 5px;
+    padding-right: 4px;
```
Comment (harshitaaaaaa) [inline Projects/Whatsapp/css/style.css:241]: Please review this commit. It's working correctly in my PC.

File: Projects/Whatsapp/css/style.css [hunk 3]
```diff
@@ -163,19 +202,19 @@
...
 
 
 #chat-text-area{
     grid-area: chat-text-area;
     display: grid;
     grid-template-columns: 1fr 1fr 1fr 1fr 1fr 5fr 1fr;
     align-content: center;
-    align-items: center;
-    
+    align-items: center;
```
Comment (harshitaaaaaa) [inline Projects/Whatsapp/css/style.css:216]: Please review this commit. It's working correctly in my PC.

File: Projects/Whatsapp/css/style.css [hunk 4]
```diff
@@ -163,19 +202,19 @@
 
 
 #ChatName{
-    font-family: monospace;
+    font-family: 'Acme', sans-serif;
```
Comment (harshitaaaaaa) [inline Projects/Whatsapp/css/style.css:204]: Please review this commit. It's working correctly in my PC.

File: Projects/Whatsapp/css/style.css [hunk 5]
```diff
@@ -123,30 +154,38 @@
...
+    font-size: 1.15rem;
+    padding-bottom: 0.2rem;
 }
+
 .chat-date{
-    font-size: 10px;
+    font-size: 14px;
 }
-.chat-message{
```
Comment (harshitaaaaaa) [inline Projects/Whatsapp/css/style.css:147]: Please review this commit. It's working correctly in my PC.

File: Projects/Whatsapp/css/style.css [hunk 6]
```diff
@@ -196,29 +235,46 @@
...
     padding: 0 5px;
+    padding-left: 6px;
     cursor: pointer;
-    }
-
-    #smiley{
-        height: 20px;
-        padding: 1px;
-        margin-top: 5px;
-        padding: 0 35px;
```
Comment (harshitaaaaaa) [inline Projects/Whatsapp/css/style.css:218]: Please review this commit. It's working correctly in my PC.

**Review by harshitaaaaaa — COMMENTED** [review]
Please review this commit. It's working correctly in my PC.

File: Projects/Whatsapp/css/style.css (hunk 1 above)
Comment (God-father1) [reply Projects/Whatsapp/css/style.css:240]: Efforts Appreciated!

**Pull request merged**